fmriprep/cli/parser.py

```diff
@@ -486,9 +486,13 @@ def _slice_time_ref(value, parser):
     g_syn = parser.add_argument_group("Specific options for SyN distortion correction")
     g_syn.add_argument(
         "--use-syn-sdc",
-        action="store_true",
+        nargs="?",
+        choices=["warn", "error"],
+        action="store",
+        const="error",
         default=False,
-        help="EXPERIMENTAL: Use fieldmap-free distortion correction",
+        help="EXPERIMENTAL: Use fieldmap-free distortion correction; "
```

Comment:
random thought - when do we plan on removing the "EXPERMENTAL" tag?

Reply:
When there's a ground truth. :-P

IDK. Maybe let's discuss at next bimonthly?